test/Traveler-test.js

```diff
@@ -0,0 +1,53 @@
+import { expect } from "chai";
+import Traveler from "../src/Traveler";
+
+describe("Traveler", () => {
+  let traveler1, traveler2, traveler3;
+  let traveler1Data, traveler2Data, traveler3Data;
+
+  beforeEach(() => {
+    traveler1Data = {
```

Comment:
Another suggestion to help clean up, you can move the test data into a mock file.  And then call on that data in each of your test files.  This helped me clean up the beforeEach, and also 1 source of data made it easier to correct changes as I updated my src files.

Reply:
If used in multiple test files, definitely worthwhile to extract. Good pattern would be to wrap it in a function and allow property overrides to be passed in so you can create multiple copies with distinct values where needed. (i.e. an instance with `id=1` and another with `id=2`).